src/gui/sdlmain.cpp

```diff
@@ -1846,19 +1950,43 @@ static void SetupWindowResolution(const char *val)
 		SDL_Rect bounds;
 		SDL_GetDisplayBounds(sdl.display_number, &bounds);
 		if (w > 0 && h > 0 && w <= bounds.w && h <= bounds.h) {
+			sdl.desktop.want_resizable_window = false;
 			sdl.desktop.window.use_original_size = false;
 			sdl.desktop.window.width = w;
 			sdl.desktop.window.height = h;
 			return;
 		}
```

Comment:
If this user's dimensions were out of bounds, suggest adding a short message indicating the user's values plus the bounds. (No need to mention falling back to `original` here, as that will be mentioned below).